doc/release-notes.md

```diff
@@ -45,15 +45,235 @@ Notable Changes
 (Developers: add your notes here as part of your pull requests whenever possible)
 
 
-Cold-Staking Re-Activation
---------------------------
-PIVX Core v6.0.0 includes a fix for the vulnerability identified within the cold-staking protocol (see PR [#2258](https://github.com/PIVX-Project/PIVX/pull/2258)).
-Therefore the feature will be re-enabled on the network, via `SPORK_19`, shortly after the upgrade enforcement.
+Deterministic Masternode Lists
+------------------------------
+
+PIVX v6.0.0 introduces on-chain consensus for masternode lists, which allow for deterministic quorum derivation, implementing Dash's [DIP-0003](https://github.com/dashpay/dips/blob/master/dip-0003.md).
+
+In the previous masternode system, each node needed to maintain their own individual masternode list with P2P messages, thus discrepancies might occur, for example, due to a different order of message reception.
+Deterministic Masternode lists are lists of masternodes, built at every block, relying only on on-chain data (previous list, and transactions included in the current block).
+All nodes derive (and verify) their masternode lists independently, from the same on-chain transactions, thus they immediately reach consensus on the tier-two state (number of masternodes, properties and status of each one).
+
+Masternodes are "registered" by special transactions called ProTx, and removed only by spending the collateral.
+A ProTx either creates a 10000-PIV collateral as tx output, or includes a reference to an unspent 10000-PIV utxo on chain (and a proof of ownership).
+See PR [#2267](https://github.com/PIVX-Project/PIVX/pull/2267) for more information.
+
+Upgrade instructions: !TODO
```

Comment:
Reminder: add the upgrade instructions.

Reply:
That's meant to be replaced with a link to a blog post and/or forum thread in the future.